### PR TITLE
feat(sqlMigration): add extra logging to the migration script

### PR DESCRIPTION
**Pull request by kevinawoo**
This is helpful for tracking the status of a migration.

### CODE REVIEW

**Comment by KathrynLewis** [comment]
Looks great! This will be really helpful 🎉

File: orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/OrchestrationMigrationAgent.kt [hunk 1]
```diff
@@ -65,10 +67,15 @@ class OrchestrationMigrationAgent(
         .toBlocking()
         .single()
 
+      log.info("Found ${unmigratedOrchestrations.size} pipelines to migrate")
+
       if (unmigratedOrchestrations.isNotEmpty()) {
         log.info("${unmigratedOrchestrations.size} orchestrations to migrate ($applicationName) [$index/${allApplications.size}]")
 
         unmigratedOrchestrations.forEach {
+          if (log.isDebugEnabled) {
```
Comment (marchello2000) [inline orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/OrchestrationMigrationAgent.kt:76]: i assume you are adding this if check for performance reasons, in which case I suggest caching the value of `log.isDebugEnabled` outside the loop (same comment for https://github.com/spinnaker/orca/pull/3626/files#diff-d6c4e5ff40be45466b0742096fd3fe0cR67)

**Comment by marchello2000** [comment]
@kevinawoo - I am curious are a lot of customers using this functionality?

File: orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/OrchestrationMigrationAgent.kt [hunk 2]
```diff
@@ -65,10 +67,15 @@ class OrchestrationMigrationAgent(
...
+            log.debug("Migrating pipeline ${it.pipelineConfigId}")
```
Comment (robzienert) [inline orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/OrchestrationMigrationAgent.kt:77]:
```suggestion
            log.debug("Migrating pipeline config ${it.pipelineConfigId}")
```

File: orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/PipelineMigrationAgent.kt [hunk 3]
```diff
@@ -57,10 +58,15 @@ class PipelineMigrationAgent(
...
         .single()
 
+      log.info("Found ${unmigratedPipelines.size} unmigrated pipelines")
+
       if (unmigratedPipelines.isNotEmpty()) {
         log.info("${unmigratedPipelines.size} pipelines to migrate ($pipelineConfigId) [$index/${allPipelineConfigIds.size}]")
 
         unmigratedPipelines.forEach {
+          if (log.isDebugEnabled) {
+            log.debug("Migrating executions for pipelineConfigId ${it.pipelineConfigId}")
```
Comment (robzienert) [inline orca-migration/src/main/kotlin/com/netflix/spinnaker/orca/pipeline/persistence/migration/PipelineMigrationAgent.kt:68]:
```suggestion
            log.debug("Migrating executions for pipeline config ${it.pipelineConfigId}")
```

**Comment by marchello2000** [comment]
@kevinawoo @KathrynLewis are you still planning on merging this PR? Please address the minor feedback and I am happy to merge it

**Pull request closed without merging**